doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
Sorry, any other rippled trusting that validator operator would need to update their validators.txt. (They actually wouldn't _have_ to remove the compromised key, since the broadcasted revocation will prevent them from actually trusting it.) However, the validator rippled itself does not need to list it's public key in validators.txt under any circumstance. It will automatically trust the key from its configured `[validator_token]`